policyengine_us/parameters/gov/states/ny/tax/income/credits/ctc/updated/older_child_amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: New York CTC amount for children ages 4-16 under updated rules
```

Comment:
Can you use active voice? and try not to use values in descriptions (in case they change in the future)
"New York provides this Child Tax Credit for each older child in the household, post 2024.

Reply:
all yaml similar format